watchdog-config/extra-sql-queries.yml

```
@@ -75,3 +75,59 @@ users_with_insificient_disk:
         usage: "LABEL"
     - cht_instance:
         usage: "LABEL"
+
+days_since_successful_purge:
+  query: |
+    SELECT 
+      purge.partner_name,
+      replace(monitoring_urls.url, 'https://', '') as cht_instance,
+      now()::date - MAX(completion_date) FILTER(WHERE error IS NULL)::date AS "days"
+    FROM
+      public.app_monitoring_purge_logs as purge
+    LEFT JOIN
+      monitoring_urls ON purge.partner_name=monitoring_urls.partner_name
+    GROUP BY
+      purge.partner_name,
+      cht_instance
+  metrics:
+    - days:
+        usage: "COUNTER"
+    - partner_name:
+        usage: "LABEL"
+    - cht_instance:
+        usage: "LABEL"
```

Prometheus [best practice](https://prometheus.io/docs/practices/naming/#base-units) is to record time values with the unit of `seconds`.  Seems reasonable to do that here (and we can always display the value in Grafana as days).  

```suggestion
last_successful_purge:
  query: |
    SELECT 
      purge.partner_name,
      replace(monitoring_urls.url, 'https://', '') as cht_instance,
      extract(epoch from (now() - MAX(completion_date) FILTER(WHERE error IS NULL))) AS "seconds"
    FROM
      public.app_monitoring_purge_logs as purge
    LEFT JOIN
      monitoring_urls ON purge.partner_name=monitoring_urls.partner_name
    GROUP BY
      purge.partner_name,
      cht_instance
  metrics:
    - seconds:
        usage: "COUNTER"
    - partner_name:
        usage: "LABEL"
    - cht_instance:
        usage: "LABEL"
```

---

Also, I think we should really consider prefixing all of these metrics with something like `dwh_...`.  A common prefect is another Prometheus [best practice](https://prometheus.io/docs/practices/naming/#metric-names) and will just make it a lot easier to maintain things going forwards....